jme3-core/src/main/java/com/jme3/material/RenderState.java

```diff
@@ -551,6 +559,10 @@ public void read(JmeImporter im) throws IOException {
         backStencilDepthPassOperation = ic.readEnum("backStencilDepthPassOperation", StencilOperation.class, StencilOperation.Keep);
         frontStencilFunction = ic.readEnum("frontStencilFunction", TestFunction.class, TestFunction.Always);
         backStencilFunction = ic.readEnum("backStencilFunction", TestFunction.class, TestFunction.Always);
+        frontStencilReference=ic.readInt("frontStencilReference",0);
+        backStencilReference=ic.readInt("backStencilReference",0);
+        frontStencilMask=ic.readInt("frontStencilMask",Integer.MAX_VALUE);
+        backStencilMask=ic.readInt("backStencilMask",Integer.MAX_VALUE);

```

Comment:
Please fix the formatting issue. (Add space before and after "=" and add space after ",")

Reply:
done

Reply:
thanks